content/pubs/pubs.bib

```diff
@@ -1,3 +1,19 @@
+@article{Wang2019,
+title = {Joint Embedding of Graphs},
+author = {Wang, Shangsi and Arroyo, Jesus and Vogelstein, Joshua T. and Priebe, Carey E.},
```

Comment:
should be Jesús (realize it's wrong on arxiv but let's be consistent w/ names in our bib file)

Reply:
done